src/lib/helpers.jsx

```diff
@@ -106,6 +106,10 @@ export const getGasPrice = () =>
       // we're only interested in gwei.
       // we round to prevent errors relating to too many decimals
       gasPrice = Math.round(gasPrice) / 10;
+
+      // sometimes the API is down, we need to return a gasprice or the dapp breaks
+      if (!gasPrice) gasPrice = 10;
```

Comment:
Maybe feathers should always return the gas price and not DApp? If not could you extract this as constant?

Reply:
The feathers API could be down as well :-)

So the dApp needs to work with a default gas price. I've made it a constant in configuration.